packages/react-redux/modules/components/configure/configure.tsx

```diff
@@ -20,57 +15,34 @@ type Props = {
   adapterArgs: AdapterArgs;
   adapter: Adapter;
 };
-type ConnectedProps = {
-  handleUpdateStatus: (status: AdapterStatus) => UpdateStatusAction;
-  handleUpdateFlags: (flags: Flags) => UpdateFlagsAction;
-};
-type State = {
-  flags: Flags;
-};
-
-const createAdapterArgs = memoize(
-  (adapterArgs: AdapterArgs, eventHandlers: AdapterEventHandlers) => ({
-    ...adapterArgs,
-    ...eventHandlers,
-  })
-);
 
-export class Configure extends React.PureComponent<
```

Comment:
I took the liberty to refactor this to a function component, as it makes the refactoring simpler